dd-sdk-android-ndk/src/androidTest/kotlin/com/datadog/android/ndk/NdkTests.kt

```diff
@@ -64,34 +81,65 @@ internal class NdkTests {
         inputStream?.use {
             val jsonString = String(it.readBytes(), Charset.forName("utf-8"))
             val jsonObject = JsonParser.parseString(jsonString).asJsonObject
-            assertThat(jsonObject).hasField("service", "")
-            assertThat(jsonObject).hasField("env", "")
+            assertThat(jsonObject).hasField("service", serviceName)
+            assertThat(jsonObject).hasField("ddtags", "env:$env")
             assertThat(jsonObject).hasField("status", "emergency")
             assertThat(jsonObject).hasField("message", "Native crash detected")
             assertThat(jsonObject).hasField("error.message", signalErrorMessage)
             assertThat(jsonObject).hasField("error.signal", "$signalName: $signal")
             assertThat(jsonObject).hasField("error.kind", "Native")
             assertThat(jsonObject).hasField("logger.name", "crash")
+            assertThat(jsonObject).hasNullableField("application_id", appId)
+            assertThat(jsonObject).hasNullableField("session_id", sessionId)
+            assertThat(jsonObject).hasNullableField("view.id", viewId)
         }
     }
 
+    // region NDK
+
     /**
      * Will run the actual test suites on the NDK side.
      * @return 0 if all the tests passed.
      */
-    private external fun runNdkUnitTests(): Int
+    private external fun runNdkSuitTests(): Int
+
+    /**
+     * Will run the singular tests on the NDK side.
+     * @return 0 if all the tests passed.
+     */
+    private external fun runNdkStandaloneTests(): Int
 
     /**
      * Will the integration test scenario for handling a crash signal on the NDK side.
      * @param storageDir the storage directory for the reported crash logs
+     * @param serviceName the service name for the main context
+     * @param environment the environment name for the main context
      * @param signal the signal id (between 1 and 32)
      * @param signalName the signal name (e.g. SIGHUP, SIGINT, SIGILL, etc.)
      * @param signalMessage the signal error message
+     * @param appId the application id for the rum context
+     * @param sessionId the session id to be passed into the rum context
+     * @param viewId the view id to be passed into the rum context
      */
     private external fun runNdkSignalHandlerIntegrationTest(
         storageDir: String,
+        serviceName: String,
+        environment: String,
+        appId: String?,
+        sessionId: String?,
+        viewId: String?,
         signal: Int,
         signalName: String,
         signalMessage: String
     )
+
+    // endregion
+
+    // region Internal
+
+    private fun randomUUIDOrNull(): String? {
+        return if (forge.aBool()) UUID.randomUUID().toString() else null
```

Comment:
You don't have to change that in this PR, but for next time, you can actually write : 
```
forge.aNullable { UUID.randomUUID() }
```